packages/ariakit/src/tooltip/__examples__/tooltip-with-placement/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { Button } from "ariakit/button";
+import { Tooltip, TooltipAnchor, useTooltipState } from "ariakit/tooltip";
+import "./style.css";
+
+export default function Example() {
+  const tooltip = useTooltipState({ placement: "bottom-end" });
```

Comment:
@diegohaz , let me know if I should add any other placement examples like `top`, `right`, etc.. or only one example is enough ?

Reply:
One example is fine I guess. :)

Reply:
made the changes